app/models/jsonrates.rb

```diff
@@ -0,0 +1,49 @@
+require 'open-uri'
+require 'json'
+
+class Jsonrates
+  ENDPOINT_API = 'http://apilayer.net/api/'
+  ACCESS_KEY = "access_key=#{ENV['ENDPOINT_API_KEY']}"
+
+  attr_accessor :action, :currencies
+  attr_reader :loaded_data
+
+  def initialize(action = :live)
+    @action = action
+    @currencies = []
+    @loaded_data = self.load_data
+  end
+
+  def load_data
+    return "Don't support this action yet!" unless check_api_support
```

Comment:
[![Documentation](https://www.pullreview.com/assets/documentation-small.png)](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-3) Add documentation to the following method[ `Jsonrates#load_data`](https://www.pullreview.com/github/fabiosammy/dev_jsonrates/reviews/issue-3)